content/2021-06-11-make-your-life-easier.md

```diff
@@ -0,0 +1,163 @@
++++
+title = "Make your life easier"
```

Comment:
It might be helpful to put a bit more context in the title, something like: Programming guidelines to make your life easier

Reply:
Good call.